meson.build

```diff
@@ -4,7 +4,8 @@ project('lite-xl',
     license : 'MIT',
     meson_version : '>= 0.56',
     default_options : [
-        'c_std=gnu11'
+        'c_std=gnu11',
+        'cpp_std=c++20'
```

Comment:
It shouldn't affect anything that isnt cpp.

I already checked their is no way to set conditional default options.

Im currently investigating if its possible to set this in the serenity cross file without breaking things

Reply:
removed it now.
It can't be added to the SerenityOS cross file because it would force the cpp_std to a higher version on ALL ports.
Meson lacks a way of setting an option at runtime, similar to `add_languages`.
So for the next update of the port `cpp_std` will be set using a `-D` flag

Reply:
At this point just keep the cpp `default_options`.
If it doesn't impact compiling our deps I guess it's alright.

Oh maybe you could add it to `cpp_args` in the `executable` command?

Reply:
no, we shouldn't.
its arguably a shortcoming of meson that we can add languages but not language options during configuration, but we can work around it by setting it via the cli.
Lite-XL is a C project at its heart and stray modules should not affect the core project options.